test/components/Layout.vue

```diff
@@ -12,23 +12,23 @@
                   :fixed-drawer="fixedDrawer"
                   class="demo-layout-transparent">
         <mdl-layout-header-row title="Title here" slot="header">
-          <div class="mdl-layout-spacer"></div>
+          <mdl-layout-spacer/>
           <!-- Navigation -->
-          <nav class="mdl-navigation">
+          <mdl-nav>
             <a class="mdl-navigation__link" href="">Link</a>
```

Comment:
I don't know if I want to use `mdl-nav-link` here or not even though I added a separate test for it.